test/liquid/combinators/lexical_token_test.exs

```diff
@@ -54,10 +53,22 @@ defmodule Liquid.Combinators.LexicalTokenTest do
   end
 
   test "range values" do
-    test_combinator("(1..10)", &Parser.range_value/1, range_value: ["(1..10)"])
-    test_combinator("(1..var)", &Parser.range_value/1, range_value: ["(1..var)"])
-    test_combinator("(var..10)", &Parser.range_value/1, range_value: ["(var..10)"])
-    test_combinator("(var1..var2)", &Parser.range_value/1, range_value: ["(var1..var2)"])
+    test_combinator("(10..1)", &Parser.range_value/1, range_value: [{:start, 10}, {:end, 1}])
```

Comment:
Can you use the keyword notation? `[{:start, 10}, {:end, 1}]` is the same as `[start: 10, end: 1]` and the latter reads better